.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,47 @@
+on:
+  push:
+    tags:
+      - "v*" # Push events to matching `v*` version srings. e.g. v1.0, v20.15.10
+
+name: Create and Publish Release
+
+jobs:
+  build:
+    name: Create and Publish Release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
You can probably get rid of `matrix` and just use `python-version: "3.x"` here?

Reply:
yep definitely.  sorry this was copy/pasted from blocks of my ci templates.  ty!

Reply:
@m4rc1e Please let me know if you want this change and want me to make the edit.